otx/cli/utils/tests.py

```diff
@@ -99,13 +64,13 @@ def otx_train_testing(template, root, otx_dir, args):
     if "--load-weights" in args:
         command_line.extend(["--load-weights", f'{os.path.join(otx_dir, args["--load-weights"])}'])
     command_line.extend(args["train_params"])
-    assert run(command_line, env=collect_env_vars(work_dir)).returncode == 0
+    assert run(command_line).returncode == 0
     assert os.path.exists(f"{template_work_dir}/trained_{template.model_template_id}/weights.pth")
     assert os.path.exists(f"{template_work_dir}/trained_{template.model_template_id}/label_schema.json")
 
 
 def otx_hpo_testing(template, root, otx_dir, args):
-    work_dir, template_work_dir, _ = get_some_vars(template, root)
+    _, template_work_dir, _ = get_some_vars(template, root)
```

Comment:
Should we rename `get_some_vars` as `get_template_dir` or something?

Reply:
And perhaps only return `template_work_dir`?

Reply:
Yeah makes sense!